### DISCUSSION TITLE
瑞典新創 DSruptive 發明「疫苗護照晶片」！

### DISCUSSION THREAD

**Discussion by yi-1111** [1101-GS4719-C/Python-Community-GS4719-C#514] [科技新知]
瑞典一家新創公司發明了一種微晶片，可以植入於皮膚下，透過掃描就可以顯示 COVID-19 疫苗護照的詳細資訊。

或許有人會認為這種侵入式的晶片植入可能還未發展成熟，因此民眾的植入意願性不高，但目前卻已經有幾千名的瑞典人選擇將晶片植入於皮膚下，這樣他們就不需要帶著鑰匙、名片、公共交通卡出門，因此才延伸出植入疫苗護照的想法。

如今，有數百萬計的人都在使用可穿戴健康記錄設備來追蹤他們的脈搏、睡眠、體溫和其他健康數據，但可穿戴設備的價格昂貴，且還會有硬體磨損導致使用壽命較短的問題，相較下來，植入晶片的安全、經濟、耐用性就顯得特別有優勢。

https://buzzorange.com/techorange/2021/12/24/implant-skin/

**Comment by minniewei** [comment]
我覺得這是未來的趨勢，之後台灣的每位公民都可能被要求要值入一片疫苗護照晶片。

**Comment by 110409538** [comment]
這項技術逐漸成熟及普及化後,我也想要嘗試看看.

**Comment by ThomasMeng0515** [comment]
之前已經有人將晶片植入手中解鎖汽車，所以感覺這真的是可行的，但是就要看大家對於這種侵入式晶片的接受度如何了

**Comment by hapiwangy** [comment]
此類侵入人體方面的科技雖然可以方便我們的生活，但另一方面也代表著有心人士也能夠好的利用、進而可能危害我們的權利。

**Comment by yuhao112** [comment]
也許是數據還不夠完善才會讓多數人有這樣的隱憂吧，等侵入人體類的裝置更穩定時，才有可能會去考慮這項技術

**Comment by sean1325418** [comment]
只是為了不想攜帶疫苗護照就植入一個晶片感覺有點不值得

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，但植入晶片非小事，植入前要三思。

**Comment by bloodtrail0511** [comment]
相信這是未來的趨勢之一，但此種侵入式的技術尚未成熟前一定要思考清楚

**Comment by awkward-willy** [comment]
未來或許人人身上都有各式各樣的晶片了
透過它們，也許也不需要攜帶各式各樣的隨身裝置
不過這些裝置直接植入身體會不會造成健康等方面的影響
還是值得我們探究與關注的

**Comment by treeann** [comment]
疫苗護照晶片感覺很實用ㄝ 希望台灣也能穎進

**Comment by Lai27330** [comment]
感覺目前我還是寧願多帶一本小黃卡，目前對晶片植入人體這項技術還有太多不確定性了。

**Comment by etin2468** [comment]
這種晶片應該可以儲存很多訊息，不過我覺得沒必要植入人體吧

**Comment by liangyinglly** [comment]
把晶片植入體內真的感覺超恐怖的，為何不使用iOS的健康app就好，畢竟手機人們也幾乎都不離身，應該蠻方便的

**Comment by leon10456** [comment]
我覺得這是未來的趨勢 感謝樓主分享

**Comment by sys511613** [comment]
將晶片植入人體雖然聽起來可怕，但如果只是像打針那樣簡單快速又不用動手術，就可以享受很多便利，我覺得我可以接受
但還是更期待生物辨識技術的發展，或許可以不用植入晶片，靠一張臉或指紋就可以儲存很多訊息

**Comment by jcpan821** [comment]
感謝分享~~

**Comment by Lynn12378** [comment]
雖然很方便，但相對的安全性好像多疑慮

**Comment by Jimmy911218** [comment]
感覺很不錯但感覺會有意協壞的影響產生
